cep-6.md

```diff
@@ -0,0 +1,215 @@
+<table>
+    <tr><td> Title </td><td>Add Channel Notifications to conda</td>
+    <tr><td> Status </td><td>Draft</td></tr>
+    <tr><td> Author(s) </td><td>Travis Hathaway &lt;thathaway@anaconda.com&gt;</td></tr>
+    <tr><td> Created </td><td> Apr 21, 2022</td></tr>
+    <tr><td> Updated </td><td> Aug 21, 2022</td></tr>
+    <tr><td> Discussion </td><td>NA</td></tr>
+    <tr><td> Implementation </td><td>NA</td></tr>
+</table>
+
+
+## Abstract
+
+In order to facilitate better communication between users of `conda` and channel owners, we want to 
+implement a notification feature.
+Specifically, this feature will give channel owners the ability to send users a small message (about the size 
+of a single Twitter post). How often this message displays will be set by channel owners
+and will be displayed for each channel a user is actively using (i.e. everything under "channels" in a
+user's `.condarc`).
+
+The messages may contain but are not limited to the following use cases:
+
+- News about the latest package additions to the channel
+- How to sponsor, volunteer or help out a particular channel
+- Warnings about a breach of TOS while using a particular channel
+- Status updates about the stability of the channel
+
+
+## Specification
+
+The specification is described below via a question-answer format.
+
+### When will this message be shown?
+
+The notification message will appear while running the following commands:
+
+- `create`
+- `env create`
+- `install`
+- `update`
+- `env update`
+- `search`
```

Comment:
```suggestion
- `create`
- `env create`
- `env update`
- `install`
- `search`
- `update`
```

Reply:
I wonder if we should also do this for `conda remove` and `conda env remove` since technically they also more like install/update than anything else.